src/libValidator/Validator.cpp

```diff
@@ -254,6 +254,8 @@ bool Validator::CheckDirBlocks(
   uint64_t totalIndex = index_num;
```

Comment:
Probably, CheckDirBlock will need mutex now. Since can be called by ProcessSetDSBlockFromSeed and also ProcessSetDirBlocksFromSeed at same time